Makefile

```diff
@@ -1,6 +1,6 @@
 NPM_PREFIX=$(realpath .)/node_modules
 PATH:=${NPM_PREFIX}/.bin:${PATH}
```

Comment:
Do we need to quote here too?

Reply:
Somehow I missed that line when I submitted the quick fix yesterday.  

After testing it appears setting either, but not both of these fixes the problem I was seeing.  This is because the SHELL variable is set to the value set in the PATH defined above (i.e. it contains the NPM_PREFIX) as this shadows the previously defined PATH.  

Given that I've moved the quoting to the line defining the PATH so we can safely reuse PATH elsewhere in the makefile if necessary.